charts/sysdig-deploy/README.md

```diff
@@ -229,146 +61,141 @@ helm install sysdig sysdig/sysdig-deploy \
+     sysdig:
+       accessKey: ACCESS_KEY
+       secureAPIToken: SECURE_API_TOKEN
+   admissionController:
+     enabled: true
+     sysdig:
+       url: URL
+   ```
 
-As a command line parameter:
-```bash
-helm install sysdig sysdig/sysdig-deploy \
-    --namespace sysdig-agent \
-    --set global.sysdig.accessKey=ACCESS_KEY \
-    --set global.sysdig.region=SAAS_REGION \
-    --set global.clusterConfig.name=CLUSTER_NAME \
-    --set global.proxy.httpProxy=PROXY_URL \
-    --set agent.proxy.httpProxy=OVERRIDE_PROXY_URL
-```
+2. Run the following:
 
-As a values file:
-```yaml
-global:
-  sysdig:
-    accessKey: ACCESS_KEY
-    region: SAAS_REGION
-  clusterConfig:
-    name: CLUSTER_NAME
-  proxy:
-    httpProxy: PROXY_URL
-
-agent:
-  proxy:
-    httpProxy: OVERRIDE_PROXY_URL
-```
+   ```console
+   helm install --namespace sysdig-agent sysdig-agent -f values.yaml sysdig/sysdig-deploy
+   ```
+
+### Example: NodeAnalyzer
 
-## NodeAnalyzer
+See the `node-analyzer` subchart [README](https://github.com/sysdiglabs/charts/tree/master/charts/node-analyzer/README.md) for the configuration values of the `nodeAnalyzer`. To override the API endpoint variable:
 
-For configuration values of the `node-analyzer`, see the `node-analyzer` subchart [README](https://github.com/sysdiglabs/charts/tree/master/charts/node-analyzer/README.md). Prefix all the specific configurations with `nodeAnalyzer.` to apply them to the chart.
+#### Using the Key-Value Pair
 
-Example: override apiEndpoint variable for nodeAnalyzer chart
+Specify each parameter using the `--set key=value[,key=value]` argument to the `helm install`command
 
-As a command line parameter:
 ```bash
 helm install sysdig sysdig/sysdig-deploy \
     --set global.sysdig.accessKey=ACCESS_KEY \
     --set agent.collectorSettings.collectorHost=COLLECTOR_ENDPOINT \
     --set nodeAnalyzer.nodeAnalyzer.apiEndpoint=API_ENDPOINT
 ```
 
-As a values file:
-```yaml
-global:
-  sysdig:
-    accessKey: ACCESS_KEY
-
-agent:
-  enabled: false
-  collectorSettings:
-    collectorHost: COLLECTOR_ENDPOINT
-
-nodeAnalyzer:
-  nodeAnalyzer:
-    apiEndpoint: API_ENDPOINT
-```
+#### Using values.yaml
 
-## KSPM Collector
+The `values.yaml` file specifies the values for the configuration parameters.  You can add the configuration to the `values.yaml` file, then use it in the `helm install` command.
 
-For configuration values of the `kspm-collector`, see the `kspm-collector` subchart [README](https://github.com/sysdiglabs/charts/tree/master/charts/kspm-collector/README.md). Prefix all the specific configurations with `kspmCollector.` to apply them to the chart.
+1. Add the following to the `values.yaml` file:
 
-Example: override apiEndpoint variable for kspmCollector chart
+   ```yaml
+   global:
+     sysdig:
+       accessKey: ACCESS_KEY
+   
+   agent:
+     enabled: false
+     collectorSettings:
+       collectorHost: COLLECTOR_ENDPOINT
+   
+   nodeAnalyzer:
+     nodeAnalyzer:
+       apiEndpoint: API_ENDPOINT
```

Comment:
This part is not needed. Since this example is not setting `global.sysdig.region=custom` it will default to `us1` and will therefore automatically populate `nodeAnalyzer.nodeAnalyzer.apiEndpoint` with the correct value.

Reply:
done